std/algorithm/setops.d

```diff
@@ -3,6 +3,17 @@
 This is a submodule of $(MREF std, algorithm).
 It contains generic algorithms that implement set operations.
 
+The functions $(LREF multiwayMerge), $(LREF multiwayUnion), $(LREF setDifference),
+$(LREF setIntersection), $(LREF setSymmetricDifference) expect a range of sorted
+ranges as input.
+
+All the algorithms, except (LREF multiwayMerge) and (LREF mutiwayUnion),
+are generalised to accept as input not only sets but also multisets:
```

Comment:
I think the common style is American English -> generalized
Grep on "generalize" at dmd/phobos found three matches, none for the British variant.